lib/tic_tac_toe_web/controllers/game_controller.ex

```diff
@@ -50,18 +59,47 @@ defmodule TicTacToeWeb.GameController do
         |> text("Oh no, that spot is already taken!")
     end
 
-    def create_move(connection, _params) do
-        data = fetch_board_update_request(connection)
-        mapped_data = json_to_map(data)
+    def mapped_request(connection, _params) do
+        fetch_board_update_request(connection)
+        |> json_to_map
+    end
+
+    def create_move(connection, params) do
+        mapped_data = mapped_request(connection, params)
+
+        case mapped_data.game_type do
+            "human_vs_human" ->
+                create_human_move(connection, params)
+            "easy_computer" ->
```

Comment:
The game type is a symbol one place and a string another...could it be made consistent or turned into constants?